devsetup/scripts/standalone.sh

```diff
@@ -80,8 +87,12 @@ scp $SSH_OPT standalone/ceph.sh root@$IP:/tmp/ceph.sh
 scp $SSH_OPT standalone/openstack.sh root@$IP:/tmp/openstack.sh
 
 # Running
-ssh $SSH_OPT root@$IP "bash /tmp/repo-setup.sh"
-# separate the two commands so we can properly detect whether there is
-# a failure while deploying standalone
-ssh $SSH_OPT root@$IP "rm -f /tmp/repo-setup.sh"
-${CLEANUP_DIR_CMD} $CMDS_FILE
+if [[ -z ${SKIP_TRIPLEO_REPOS} || ${SKIP_TRIPLEO_REPOS} == "false" ]]; then
+    ssh $SSH_OPT root@$IP "bash /tmp/repo-setup.sh"
```

Comment:
In this pr, we have added two vars $REPO_SETUP_CMDS and $CMDS_FILE.
$REPO_SETUP_CMDS will be used in RDO upstream zuul jobs to enable repos using triple repos and content from $REPO_SETUP_CMDS via repo-setup.sh.

$CMDS_FILE will be used in downstream zuul jobs, to enable custom rhel repos via standalone-deploy.sh

and if we set SKIP_TRIPLEO_REPOS to true then, it will be skipped.

Looks good.

Can we initialize `SKIP_TRIPLEO_REPOS=''"` so that this if conditions always run for upstream and it will avoid to set extra var in the job definition?